### ISSUE TITLE
What is the training dataset used in the paper?

### ISSUE THREAD

**Issue by rxlian** [amazon-research/sccl#30]
I think the performance was evaluated on various short text clustering datasets. I was wondering what is the training dataset used in this paper?

**Comment by urospet** [comment]
@rxlian They seem to have use the same training and testing set. Trained a model per dataset.

**Comment by rxlian** [comment]
> @rxlian They seem to have use the same training and testing set. Trained a model per dataset.

So the clustering loss was trained in a supervised manner?
